assessments/projects/openfga/joint-assessment.md

```diff
@@ -0,0 +1,647 @@
+- Checksum.sig/pem
+- in-toto
+
+#### User Instructions for Validating Artifact Signatures
+
+| Aspect                          | Status |
+|---------------------------------|--------|
+| Clear instructions are provided | Yes    |
+| Instructions are maintained independently from the release process. | [OpenFGA Provenance Implementation](https://github.com/openfga/community/blob/main/provenance-implementation/openfga.md) |
+
+### Reporting Security Incidents
+
+#### Security Reporting Method
+| Aspect                     | Status |
+|----------------------------|--------|
+| Documentation Location     | Project’s Security tab                              |
+| Email                      | [security@openfga.dev](mailto:security@openfga.dev) |
+| SLA                        | 5-day                                               |
+| Visibility Test Response   | 2 minutes                                           |
+
+#### Security Reporting Ownership
+
+| Aspect                        | Status                                     |
+|-------------------------------|--------------------------------------------|
+| Bus-Proofing                  | Checked by at least 2 users, available to more |
+| Layoff-Proofing               | Not implemented; core maintainers are all currently from Okta |
+
+### Reducing Vulnerabilities Through Development
+
+| Aspect | Details |
+|--------|---------|
+| Secure Development Practices | Optional secure development training is provided by Okta. | Security Review is done for every feature addition. |
+| Code Quality and Testing | CodeQL is used on every pull request. The team is confident in the test coverage. |
+| Binary Management | CLOMonitor check passes, and the team is aware of the dangers of allowing binaries in the project. |
+| OpenSSF Scorecard | Badge present. Score is 9.3, well above the average of 4. |
+| OpenSSF Best Practices | Badge present. Passing grade. |
+| CLOMonitor | 100% security score. |
+
+### Dependency Management
+
+| Aspect                      | Status |
+|-----------------------------|--------|
+| Lifecycle Policy            | [Dependency Lifecycle Policy](https://github.com/openfga/openfga/blob/main/docs/dependencies-policy.md) |
+| SCA Checks                  | - [Semgrep Workflow](https://github.com/openfga/openfga/blob/main/.github/workflows/semgrep.yaml)<br>- [Pull Request Workflow](https://github.com/openfga/openfga/blob/main/.github/workflows/pull_request.yaml)<br>- Dependabot |
+| SCA Findings Review Process | Recommendations from Dependabot, Snyk, etc., are reviewed during a weekly meeting |
+
+### Security Champions
+
+The following maintainers take a special interest in project security:
+
+- Louis Jette
+- Maria Inesparnisari
+
+### Roadmapped Security Improvements
+
+- Increase OpenSSF Best Practices badge level.
+- Implement a singleflight CheckResolver to avoid concurrency evaluation of overlapping subproblems. [#1301](https://github.com/openfga/openfga/issues/1301)
```

Comment:
```suggestion
- Implement a singleflight CheckResolver to avoid concurrent evaluation of overlapping subproblems. [#1301](https://github.com/openfga/openfga/issues/1301)
```